engine/schema/src/com/cloud/upgrade/DatabaseUpgradeChecker.java

```diff
@@ -120,7 +121,7 @@ public DatabaseUpgradeChecker() {
                 new Upgrade410to420(), new Upgrade420to421(), new Upgrade421to430(), new Upgrade430to440(), new Upgrade440to441(), new Upgrade441to442(), new Upgrade442to450(),
                 new Upgrade450to451(), new Upgrade451to452(), new Upgrade452to460(), new Upgrade460to461(), new Upgrade461to470(), new Upgrade470to471(), new Upgrade471to480(),
                 new Upgrade480to481(), new Upgrade481to490(), new Upgrade490to4910(), new Upgrade4910to4920(), new Upgrade4920to4930(), new Upgrade4930to41000(),
-                new Upgrade41000to41100()});
+                new Upgrade41000to41100(), new Upgrade41100to41110()});
```

Comment:
@ernjvr, Thanks for the PR. Would be good if you add a unit test to find the upgrade path from 41100 to 41110 in this file (engine/schema/test/com/cloud/upgrade/DatabaseUpgradeCheckerTest.java).